### PR TITLE
metrics(plots): regular udpates & terminology

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Improve metrics/plots explanations
- "metric file" -> "metrics file" per #1442 and a checkbox in #608

### CODE REVIEW

File: content/docs/command-reference/metrics/diff.md [hunk 1]
```diff
@@ -21,23 +21,18 @@ positional arguments:
 ## Description
 
 This command provides a quick way to compare metrics among experiments in the
-repository history. It requires that Git is being used to version the metrics.
+repository history. All metrics defined in `dvc.yaml` are used by default. The
+differences shown by this command include the new value, and numeric difference
+(delta) from the previous value of metrics (rounded to 5 digits precision).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/diff.md:26]:
```suggestion
comparison shown by this command includes the old and new value, and the
numeric difference (delta, rounded to 5 digits precision).
```

File: content/docs/command-reference/metrics/diff.md [hunk 2]
```diff
@@ -21,23 +21,18 @@ positional arguments:
 ## Description
 
 This command provides a quick way to compare metrics among experiments in the
-repository history. It requires that Git is being used to version the metrics.
+repository history. All metrics defined in `dvc.yaml` are used by default. The
+comparison shown by this command includes the old and new value, and the
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:25]: are you sure we include old?

File: content/docs/command-reference/metrics/index.md [hunk 3]
```diff
@@ -32,8 +32,9 @@ ability to mark a certain stage <abbr>outputs</abbr> as metrics. These metrics
 are project-specific floating-point or integer values e.g. AUC, ROC, false
 positives, etc.
 
-This kind of metrics can be defined with the `-m` (`--metrics`) and `-M`
-(`--metrics-no-cache`) options of `dvc run`.
+This type of metrics files are typically generated by user data processing code,
+anf get defined with the `-m` (`--metrics`) and `-M` (`--metrics-no-cache`)
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:36]:
```suggestion
and are tracked using the `-m` (`--metrics`) and `-M` (`--metrics-no-cache`)
```

File: content/docs/command-reference/metrics/show.md [hunk 4]
```diff
@@ -19,9 +19,6 @@ positional arguments:
 Finds and prints all metrics in the <abbr>project</abbr> by examining all of its
 [DVC-files](/doc/user-guide/dvc-files-and-directories).
 
-> This kind of metrics can be defined with the `-m` (`--metrics`) and `-M`
-> (`--metrics-no-cache`) options of `dvc run`.
-
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:24]: Let this note only in the index. Should be enough

File: content/docs/command-reference/plots/diff.md [hunk 5]
```diff
@@ -21,10 +21,10 @@ positional arguments:
 
 This command is a way to visualize the "difference" between metrics among
 experiments in the <abbr>repository</abbr> history, by plotting multiple
-versions of the metrics. All plots defined in `dvc.yaml` are used by default.
+versions of them. All plots defined in `dvc.yaml` are used by default.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/diff.md:24]: Rolling this one back
```suggestion
versions of the metrics. All plots defined in `dvc.yaml` are used by default.
```

File: content/docs/command-reference/plots/diff.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ experiments in the <abbr>repository</abbr> history, by plotting multiple
 versions of the metrics. All plots defined in `dvc.yaml` are used by default.
 
 > Note that unlike `dvc metrics diff`, this command does not calculate numeric
-> differences between metric file values.
+> differences between metrics file values.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/diff.md:27]: - [ ] Need to apply this everywhere, actually.

File: content/docs/command-reference/metrics/diff.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/diff.md:25]: No. This is wrong actually, thanks. Fixing:
```suggestion
comparison shown by this command includes the new value, and the
```

File: content/docs/command-reference/metrics/diff.md [hunk 7]
```diff
@@ -21,23 +21,18 @@ positional arguments:
 ## Description
 
 This command provides a quick way to compare metrics among experiments in the
-repository history. It requires that Git is being used to version the metrics.
+repository history. All metrics defined in `dvc.yaml` are used by default. The
+comparison shown by this command includes the new value, and the
+numeric difference (delta, rounded to 5 digits precision).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/diff.md:26]:
```suggestion
comparison shown by this command includes the new value, and the numeric
difference (delta) with the previous value (rounded to 5 digits precision).
```

File: content/docs/command-reference/plots/diff.md (hunk 6 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/plots/diff.md:27]: Applied metric->metrics (file) everywhere in docs now!

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/metrics/show.md [hunk 8]
```diff
@@ -9,8 +9,8 @@ usage: dvc metrics show [-h] [-q | -v] [-a] [-T] [--all-commits] [-R]
                         [--show-json] [targets [targets ...]]
 
 positional arguments:
-  targets               Limit command scope to these metric files.
-                        Using -R, directories to search metric files
+  targets               Limit command scope to these metrics files.
+                        Using -R, directories to search metrics files
                         in can also be given.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:14]: metrics show

File: content/docs/command-reference/plots/modify.md [hunk 9]
```diff
@@ -12,7 +12,7 @@ usage: dvc plots modify [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
                         target
 
 positional arguments:
-  target                Metric file to set properties to
+  target                Metrics file to set properties to
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/modify.md:15]: plots modify

File: content/docs/command-reference/plots/show.md [hunk 10]
```diff
@@ -12,7 +12,7 @@ usage: dvc plots show [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
                       [targets [targets ...]]
 
 positional arguments:
-  targets               Metric files to visualize.
+  targets               Metrics files to visualize.
                         Shows all plots by default.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:16]: plots show

**Review by jorgeorpinel — COMMENTED** [review]
- [x] A few of these would imply a change in the core repo too:

**Pull request merged**